### DISCUSSION TITLE
#include "mbedtls/ssl.h" ?

### DISCUSSION THREAD

**Discussion by Tiese** [espressif/arduino-esp32#9940] [Q&A]
Hello!
If I update from version 2.0.17 to version 3.0.1, the error...
`fatal error: mbedtls/net.h: No such file or directory
   36 | #include "mbedtls/net.h"`
...appears when compiling!
What can I do now?

Thank you for your help!

**Comment by me-no-dev** [comment]
You are not giving much to go by. but it looks like you are trying to include a file that is no longer present.

**Reply by Tiese** [reply]
Thank you for the quick reply! 👍 
 Yes, that's correct.
 But why isn't it the same in version 2.0.17?

**Reply by me-no-dev** [reply]
mbedtls version has been updated in ESP-IDF and changes came from there

**Reply by Tiese** [reply]
That's fine,
but what can I do?
I have installed the library from: [https://github.com/Seeed-Studio/Seeed_Arduino_mbedtls](https://github.com/Seeed-Studio/Seeed_Arduino_mbedtls).
That didn't help!

**Reply by me-no-dev** [reply]
Arduino already comes with MbedTLS from ESP-IDF. You need to find the correct header that exposes the functionality you would be missing.

**Reply by Tiese** [reply]
**Thank you for your help!**
I'm just working with version 2.0.17!

**Comment by Tiese** [comment]
I didn't get any help, but it was worth a try!

**Comment by brodrigz** [comment]
Also started getting this error when upgrading from 2.X to 3.X, was this a breaking change?
